packages/story-editor/src/components/form/color/color.js

```diff
@@ -52,7 +52,8 @@ const containerCss = css`
 const Container = styled.section`
   ${containerCss}
   gap: ${({ isInDesignMenu }) => (isInDesignMenu ? 6 : 8)}px;
-  width: ${({ width }) => (width ? `${width}px` : `100%`)};
+  width: ${({ width, ignoreWidth }) =>
+    (ignoreWidth && 'inherit') || (width ? `${width}px` : `100%`)};
```

Comment:
If the `containerCss` has `display: flex` then I think `width: 100%` is redundant since flexbox will stretch the section to fill the parent too?